### ISSUE TITLE
SignalR Hub on IIS

### ISSUE THREAD

**Issue by blenet** [JustMaier/angular-signalr-hub#20]
Hi, 

it's me again,  i am using your angular signalr hub a couple of weeks and so far everything 
worked fine but now i ran into a problem, maybe it's a general signalR roblem or a config thing, i don't know right now. 
I was hosting my testapp on the ASP development Server where everyting worked fine, now i was trying to host it on my local IIS 8.5 and it's not working anymore. I receive an error while starting the hub ("Error during negotiation request -> Page not Found 404"), therefore i have also tested your sample Grid on my local IIS and i received the same error. Any ideas what i am doing wrong, i read a lot about crossdomain access and stuff like that, but i dont think that this is the problem.

Kind Regards
Manuel

**Comment by houseofbob** [comment]
I'm curious if there's been any headway on this issue.  I'm running into the same thing on Angular 1.3.10.

Everything seems to work fine if you're debugging inside VisualStudio (2012 and 2013), but the moment it is deployed to an actual IIS - whether localhost or remote - this issue crops up.  According to my browser dev tools, under the networking tab, there seems to be looking for https://localhost/signalr and not finding it.  

Why is it looking for such a folder at the root of the server, instead of inside the server? Why does this error not come up when debugging in VS? And more importantly, how can we work around this?

**Comment by houseofbob** [comment]
I just realized I forgot to update my solution.  After taking another look at your documentation, I saw the rootpath attribute in the hub factory, and was able to set that to my desired path of the signalr folder. Thank you for adding that attribute to your wrapper!!!

I've successfully launched an application with angular 1.3.10 and your library on windows server 2012 running iis 8.5.  Given the rootpath attribute is in place to handle just this type of situation, I'd consider this issue closed.
